pkg/kyverno/apply/Readme.md

```diff
@@ -0,0 +1,134 @@
+Apply Command - Valid Parameter Combinations and their interpretation!
```

Comment:
@JimBugwadia Since we created website to maintain the docs, where should we update Readme ?